packages/client/src/runtime/RequestHandler.ts

```diff
@@ -70,6 +70,7 @@ function getRequestInfo(request: Request) {
 
   return {
     batchTransaction: transaction?.kind === 'batch' ? transaction : undefined,
+    interactiveTransaction: transaction?.kind === 'itx' ? transaction : undefined,
```

Comment:
If we need both now, why don't just pass `transaction` as it is and differentiate by `kind` field? That was my original intention with this type.